template/Dockerfile

```diff
@@ -17,7 +17,9 @@
 # Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 # 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 # ---------------------------------------------------------------------------
-FROM tomcat:8-jdk8-slim
+
+<% if (image == 'debian') print 'FROM tomcat:8-jdk8-slim'
+   else if (image == 'alpine') print 'FROM tomcat:8-jre8-alpine' %>
```

Comment:
I'm wondering if using a jre instead of a jdk wouldn't cause trouble.

Reply:
I have tested the app was working fine, but if you have other concerns I would try to change it?

Reply:
Basically we were using jar and war files. So I thought there is no need for JDK as we don't want to compile anything. That's why I choose JRE.

Reply:
Here should be sufficient to run a Java Webapp. The package and image is smaller too.

Reply:
JRE should be sufficient to run Java apps. The package and image layer will be smaller too.

Reply:
JRE should be fine, yes.

Reply:
All variants schould use a jre to run the software.

Reply:
If only a JRE is needed (still not sure it's the case and there might be some 3rd party lib we use that would require a JDK. As an example, back in the days - don't know if it's still the case or not -,  if you use JSPs you needed a JDK since it had to compile the JSP to java code), then indeed we should change all images. We also need to update https://dev.xwiki.org/xwiki/bin/view/Community/SupportStrategy/JavaSupportStrategy/ to explain that only a JRE is needed for running XWiki.

Reply:
The XWiki Debian package depends on `openjdk-11-jre-headless` so yes, the JRE is definitely enough, and it's like this since we have Debian packages.